xclim/testing/tests/test_formatting.py

```diff
@@ -33,3 +37,23 @@ def test_indicator_docstring():
 
     doc = degree_days_exceedance_date.__doc__.split("\n")
     assert doc[20] == "  Default : >. "
+
+
+def test_update_xclim_history(atmosds):
+    @fmt.update_xclim_history
+    def func(da, arg1, arg2=None, arg3=None):
+        return da
+
+    out = func(atmosds.tas, 1, arg2=[1, 2], arg3=None)
+
+    matches = re.match(
+        r"\[([0-9-:\s]*)\]\s(\w*):\s(\w*)\((.*)\)\s-\sxclim\sversion:\s([0-9\.]*)",
```

Comment:
We might want to grab `beta` or `alpha` or whatever.
```suggestion
        r"\[([0-9-:\s]*)\]\s(\w*):\s(\w*)\((.*)\)\s-\sxclim\sversion:\s\d*\.\d*\.\d*?\-\w*",
```

Reply:
Indeed, thanks!